cardano-cli/src/Cardano/CLI/Shelley/Orphans.hs

```diff
@@ -105,31 +106,36 @@ deriving newtype instance ToJSON (MetaDataHash era)
 deriving newtype instance ToJSON Ledger.LogWeight
 deriving newtype instance ToJSON Ledger.Likelihood
 deriving newtype instance ToJSON (Ledger.Stake StandardShelley)
+deriving newtype instance ToJSON (Ledger.PoolDistr StandardCrypto)
+deriving newtype instance ToJSON DeltaCoin
```

Comment:
The choice between `newtype` and `anyclass` determines whether the constructor name is included in the output or not, right? If the format of the ledger dump is important, then somebody should review this (and the other new instances below) more carefully.

Note that `PoolDistr` currently doesn't have a `Generic` instance, so we can't use `anyclass` for it.